R/percent-time-canada.R

```diff
@@ -65,10 +65,10 @@
 pct_time_fun <-
   function(DHHGAGE_cont, SDCGCBG, SDCGRES) {
     if (is_equal(SDCGCBG, 1)) {
-      return(1)
+      return(100)
```

Comment:
This seems a bit more then a minor change no? Document change from 0-1 as 0-100%